packages/react-instantsearch/src/components/Menu.js

```diff
@@ -76,4 +76,9 @@ class Menu extends Component {
 export default translatable({
   showMore: extended => extended ? 'Show less' : 'Show more',
   noResults: 'No Results',
+  submit: null,
+  reset: null,
+  resetTitle: 'Clear the search query.',
```

Comment:
It was added like that " * @themeKey ais-Menu__SearchBox - the container of the search for facet values searchbox. See [the SearchBox documentation](widgets/SearchBox.html#classnames) for the classnames and translation keys of the SearchBox.
" but I can add them completely instead if you think that's better.